### PR TITLE
Update changelog for 0.2.0

### PR DESCRIPTION
**Pull request by robinlinden**
---

This change is [Reviewable](https://reviewable.io/reviews/toktok/c-toxcore/811)

### CODE REVIEW

**Comment by iphydf** [comment]
We can wait with this one for #295.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,150 @@
...
+### Closed issues:
+
+- [#704](https://github.com/TokTok/c-toxcore/issues/704) Add CORRECTION support to group chats
+- [#620](https://github.com/TokTok/c-toxcore/issues/620) WIP: Video bug found!
+- [#606](https://github.com/TokTok/c-toxcore/issues/606) groupId is int whereas friendId is uint32_t, reason?
+- [#572](https://github.com/TokTok/c-toxcore/issues/572) int32_t may be not large enough as a argument for video_bit_rate of vp8/9 codec
+- [#566](https://github.com/TokTok/c-toxcore/issues/566) LAYER #: modules for static linking - build issue
+- [#42](https://github.com/TokTok/c-toxcore/issues/42) Remove ToxDNS and related stuff from toxcore
+
+## v0.1.12
```
Comment (zoff99) [inline CHANGELOG.md]: 0.1.12 was never released i think

Comment (robinlinden) [reply CHANGELOG.md]: Yeah, there's no 0.1.12 tagged, but this is autogenerated from our milestones. I guess those two items should be moved to the 0.2.0 milestone. I'll have to regenerate this after the persistent conferences are merged, so I'll sort it out then.

**Comment by iphydf** [comment]
@robinlinden everything that needs to be in 0.2.0 is merged. We can go ahead with this now.

File: CHANGELOG.md (hunk 1 above)
Comment (nurupo) [reply CHANGELOG.md]: Sounds like the persistent conferences will not make it in 0.2.0, so you don't have to wait on them being merged to sort this out.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,150 @@
...
+- [#701](https://github.com/TokTok/c-toxcore/pull/701) Add astyle to Circle CI build.
+- [#700](https://github.com/TokTok/c-toxcore/pull/700) Use more descriptive names in bwcontroller.
+- [#699](https://github.com/TokTok/c-toxcore/pull/699) Add some explanatory comments to the toxav audio code.
+- [#698](https://github.com/TokTok/c-toxcore/pull/698) Extract named constants from magic numbers in toxav/audio.c.
+- [#697](https://github.com/TokTok/c-toxcore/pull/697) Use C99 standard in bazel builds.
+- [#694](https://github.com/TokTok/c-toxcore/pull/694) Add bazel build scripts for c-toxcore.
+- [#693](https://github.com/TokTok/c-toxcore/pull/693) Make libcheck optional for windows builds.
+- [#691](https://github.com/TokTok/c-toxcore/pull/691) Don't install packages needlessly on Travis
+- [#690](https://github.com/TokTok/c-toxcore/pull/690) Run fewer Travis jobs during Pull Requests.
+- [#689](https://github.com/TokTok/c-toxcore/pull/689)  Make Net_Crypto a module-private type.
```
Comment (nurupo) [inline CHANGELOG.md]: Two spaces.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,150 @@
...
+- [#578](https://github.com/TokTok/c-toxcore/pull/578) Split toxav_bit_rate_set() into two functions to hold the maximum bitrates libvpx supports
+- [#477](https://github.com/TokTok/c-toxcore/pull/477) Update install instructions to use CMake
+- [#465](https://github.com/TokTok/c-toxcore/pull/465) Add Alpine linux Dockerfile in addition to the existing Debian one
+- [#442](https://github.com/TokTok/c-toxcore/pull/442) Generate only one large library "libtoxcore".
+- [#334](https://github.com/TokTok/c-toxcore/pull/334) Change toxencryptsave API to never overwrite pass keys.
+
+### Closed issues:
+
+- [#704](https://github.com/TokTok/c-toxcore/issues/704) Add CORRECTION support to group chats
+- [#620](https://github.com/TokTok/c-toxcore/issues/620) WIP: Video bug found!
```
Comment (nurupo) [inline CHANGELOG.md]: The issue title probably should be changed so it doesn't have "WIP" in it?

File: CHANGELOG.md (hunk 1 above)
Comment (nurupo) [reply CHANGELOG.md]: Here is the citation for the persistent conferences not making it in: https://github.com/TokTok/c-toxcore/issues/810.

**Comment by iphydf** [comment]
@nurupo all done.

**Comment by iphydf** [comment]
:lgtm_strong:

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/c-toxcore/811#-:-L6OvDF7cnP05I2ZyvRb:b-jbsf8z)*

File: CHANGELOG.md [hunk 4]
```diff
@@ -331,7 +475,7 @@
 
 ### Merged PRs:
 
-- [#207](https://github.com/TokTok/c-toxcore/pull/207) docs: correct instructions for cloning & harden against repo name changes
+- [#207](https://github.com/TokTok/c-toxcore/pull/207) docs: correct instructions for cloning & harden agains repo name changes
```
Comment (iphydf) [inline CHANGELOG.md]: Not fixing this now. We can fix it in the next release.

**Pull request merged**